### PR TITLE
Bugfix: project responsive

### PR DESCRIPTION
**Pull request by giancorzo**
https://developertown.visualstudio.com/SIL%20International%20Scriptoria/_workitems/edit/29694

project-1
product-2
product-3

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
screenshots?

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/project.scss [hunk 1]
```diff
@@ -1,164 +1,17 @@
-.project-details {
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/project.scss]: big fan of stuff getting deleted :+1:

**Comment by NullVoxPopuli** [comment]
what's it look like on desktop-sized screens?

**Comment by giancorzo** [comment]
> what's it look like on desktop-sized screens?

project-desktop-1

**Comment by giancorzo** [comment]
Ipad view

project-desktop-1

**Pull request merged**